### ISSUE TITLE
Note that Xonsh require Python 3 for attendees.

### ISSUE THREAD

**Issue by Carreau** [xonsh/tutorial#1]
Cause installing with pip is fast unless you don't have Python 3.

**Comment by gforsyth** [comment]
Agreed on this, but I don't think it needs to be in the tutorial outline / abstract.  Tutorial leads are given a mailing list of all attendees so they can send any pre-tutorial instructions and the Python 3 requirement can be sent out then.
